posthog/utils.py

```diff
@@ -187,6 +187,19 @@ def get_git_commit() -> Optional[str]:
         return None
 
 
+def get_git_sha() -> Optional[str]:
```

Comment:
This is the same as `get_git_commit`, only without arg `--short`, so I'd suggest just modifying that existing function by changing its signature to `def get_git_commit(short: bool = True) -> Optional[str]:`